runtests.py

```diff
@@ -474,23 +474,19 @@ def build_project(args):
             '--single-version-externally-managed',
             '--record=' + dst_dir + 'tmp_install_log.txt']
 
-    py_v_s = sysconfig.get_config_var('py_version_short')
-    platlibdir = getattr(sys, 'platlibdir', '')  # Python3.9+
+    config_vars = dict(sysconfig.get_config_vars())
+    config_vars["platlibdir"] = getattr(sys, 'platlibdir', '')  # Python3.9+
```

Comment:
```suggestion
```
Do we even need this?  If it isn't there, presumably the `get_path` doesn't use it?